test/unit/ngsi-ld/examples/contextRequests/updateContextCommandStatus1.json

```diff
@@ -0,0 +1 @@
+[{"@context":"http://context.json-ld","id":"urn:ngsi-ld:Robot:r2d2","type":"Robot","position_status":{"type":"Property","value":{"@type":"commandStatus","@value":"UNKNOWN"}},"position_info":{"type":"Property","value":{"@type":"commandResult","@value":" "}}}]
```

Comment:
Better to have the .json in pretty printed form, as the others.

Reply:
Fixed:  cac4d5f